Installation instructions for @humanmade/eslint-config are incorrect

I'm trying to set up a fresh project which uses `@humanmade/eslint-config` for its linting. The installation instructions at https://github.com/humanmade/coding-standards/blob/master/packages/eslint-config-humanmade/readme.md appear to be incorrect as I only get errors after following them.

## Steps to reproduce:

1. Create a new JS project with no existing dependencies
2. Add some JS files to a `src` directory
3. Follow the setup instructions at https://github.com/humanmade/coding-standards/blob/master/packages/eslint-config-humanmade/readme.md
4. Add a `.eslintrc.json` file containing the following:  

       {
         "extends": "humanmade",
       }


4. Run `./node_modules/.bin/eslint -c humanmade src`

Observe the following error message:

```
Error: Cannot read config file: /Users/john/projects/projectname/humanmade
```

I found #176 which looks like the `humanmade` package is now `@humanmade/eslint-config` so I changed the `extends` property to that and now I get the following error:

```
Error: Cannot find module 'eslint-plugin-jsdoc'
```

Am I doing something wrong or are the installation instructions incorrect?

--- a/packages/eslint-config-humanmade/readme.md
+++ b/packages/eslint-config-humanmade/readme.md
@@ -4,10 +4,7 @@ Human Made coding standards for JavaScript.
 
 ## Installation
 
-This package is an ESLint shareable configuration, and requires:
-
-
-`babel-eslint`, `eslint`, `eslint-config-react-app`, `eslint-plugin-flowtype`, `eslint-plugin-import`, `eslint-plugin-jsx-a11y`, `eslint-plugin-jsdoc`, `eslint-plugin-react`, `eslint-plugin-react-hooks`.
+This package is an ESLint shareable configuration, and requires: `babel-eslint`, `eslint`, `eslint-config-react-app`, `eslint-plugin-flowtype`, `eslint-plugin-import`, `eslint-plugin-jsx-a11y`, `eslint-plugin-jsdoc`, `eslint-plugin-react`, `eslint-plugin-react-hooks`.
 
 To install this config and the peerDependencies when using **npm 5+**:
 

--- a/packages/eslint-config-humanmade/readme.md
+++ b/packages/eslint-config-humanmade/readme.md
@@ -4,7 +4,10 @@ Human Made coding standards for JavaScript.
 
 ## Installation
 
-This package is an ESLint shareable configuration, and requires `babel-eslint`, `eslint`, `eslint-config-react-app`, `eslint-plugin-flowtype`, `eslint-plugin-import`, `eslint-plugin-jsx-a11y`, `eslint-plugin-react`.
+This package is an ESLint shareable configuration, and requires:
+
+
+`babel-eslint`, `eslint`, `eslint-config-react-app`, `eslint-plugin-flowtype`, `eslint-plugin-import`, `eslint-plugin-jsx-a11y`, `eslint-plugin-jsdoc`, `eslint-plugin-react`, `eslint-plugin-react-hooks`.
 
 To install this config and the peerDependencies when using **npm 5+**:
 
@@ -16,21 +19,21 @@ npx install-peerdeps --dev @humanmade/eslint-config@latest
 
 You can then use it directly on the command line:
 
-```
-eslint -c humanmade MyFile.js
+```shell
+./node_modules/.bin/eslint -c @humanmade/eslint-config MyFile.js
 ```
 
 Alternatively, you can create your own configuration and extend these rules:
 ```yaml
 extends:
-- humanmade
+- @humanmade/eslint-config
 ```
 
 ### Working with TypeScript
 
 If you desire to use TypeScript for your project, you will need to add another dependency:
 
-```
+```shell
 npm install --save-dev @typescript-eslint/parser
 ```
 
@@ -39,7 +42,7 @@ Once it's installed, update your configuration with the `parser` parameter:
 ```yml
 parser: "@typescript-eslint/parser"
 extends:
-    - humanmade
+    - @humanmade/eslint-config
 ```
 
 ## Global Installation
@@ -48,7 +51,7 @@ When installing globally, you need to ensure the peer dependencies are also inst
 
 Run the same command as above, but instead with `--global`:
 
-```
+```shell
 npx install-peerdeps --global @humanmade/eslint-config@latest
 ```
 

